src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -96,27 +97,33 @@ class TFBertPreTrainingLoss:
     computation.
     """
 
-    def compute_loss(self, labels, logits):
+    def compute_loss(self, labels: tf.Tensor, logits: tf.Tensor) -> tf.Tensor:
         loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(
             from_logits=True, reduction=tf.keras.losses.Reduction.NONE
         )
         # make sure only labels that are not equal to -100
         # are taken into account as loss
-        masked_lm_active_loss = tf.not_equal(tf.reshape(labels["labels"], (-1,)), -100)
+        masked_lm_active_loss = tf.not_equal(x=tf.reshape(tensor=labels["labels"], shape=(-1,)), y=-100)
```

Comment:
I'm all for using keywords in keywords argument, but here, `x` and `y` are not very informative, so it makes the code less readable in this instance. I would leave as is for this part.

Reply:
Here in particular, the `x` and `y` don't help readability at all imo